### ISSUE TITLE
Unable to add database

### ISSUE THREAD

**Issue by abhinav1293** [d4software/QueryTree#1]
Test Database connection is not working. when I entered the  mysql details and click on test connection nothing happen.

thanks you

**Comment by PandelisZ** [comment]
Do you definately have access to the database from where you are running query tree? 

If it's an auth error we should probably show that error on our side too however

**Comment by abhinav1293** [comment]
Yes, I do have an access to the database.  In fact, I have tested with another method just to check auth credential working or not. It is working fine. 

By clicking on Test connection it is not showing any message (Either success or failure).

**Comment by dand4** [comment]
It's interesting that you're saying the Test Connection button doesn't show any message. I've just been testing 03cdf7a with a variety of bad configs (e.g. wrong username, wrong server host, wrong SSH user) they all result in an error coming back and being shown.

The fact that nothing is shown suggests that it's still trying, but it's taking a very long time. 

Are you using SSH? If so, are you connecting with a keyfile or password?

**Comment by abhinav1293** [comment]
Actually, I am not using SSH to connect Mysql. just username and password of MySQL server.

**Comment by dand4** [comment]
I wonder if perhaps your firewall is not allowing QueryTree to open a connect to the database server/port. But the other application that you tested your connection settings with is allowed?

**Comment by leejern** [comment]
I've added MySQL connection, tested it, and it is okay. But the tables are not showing. it has been loading for like 1 hour
